code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -45,7 +45,13 @@
 	w_class = SIZE_SMALL
 	force = MELEE_FORCE_WEAK
 	var/on = 0
+	var/stun_time = 1 SECONDS
 
+/obj/item/weapon/melee/telebaton/attack(mob/living/target, mob/living/user)
+	if(!on || user.a_intent == INTENT_HARM)
```

Comment:
```suggestion
	if(!on || user.a_intent == INTENT_HARM || isyautja(target))
```
because I can already see ~~myself~~ people abusing this to stun preds